UI.tsv

```diff
@@ -1498,13 +1498,13 @@ UI_20150414_001497	Roasting Level %s
 UI_20150428_001498	{@st43}Buy-in Shop{/}
 UI_20150428_001499	{@st41b}Close
 UI_20150428_001500	{@st41b}Bug Report
-UI_20150428_001501	{@st59}Notify about problem{/}
-UI_20150428_001502	Your game play until now will be sent when you make a bug report. The replay file will be of much help in fixing the bug. Do you wish to send it?
+UI_20150428_001501	{@st59}Notify us about bugs and issues{/}
+UI_20150428_001502	When you report a bug, your current gameplay screen will be sent to our server.{nl}The replay file will be of much help in resolving the bug. Do you want to send it?
```

Comment:
I don't know what you mean by "current gameplay screen" then if you didn't mean screenshot.